src/app.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const fileCopy = (src, dest) => {
+  if (src === dest || (!src || !dest)) {
```

Comment:
The condition block should throw an error or return a meaningful message when the function parameters are invalid.